lib/velum/salt.rb

```diff
@@ -41,6 +41,26 @@ def self.minions
       JSON.parse(res.body)["return"].first
     end
 
+    # Returns the minions that have not been accepted into the cluster
+    def self.pending_minions
+      res = perform_request(endpoint: "/", method: "post",
+                            data: { client: "wheel",
+                                    fun:    "key.list" ,
+                                    match:  "all",
+                                    tgt:    "ca" })
+      JSON.parse(res.body)["return"].first["data"]["return"]["minions_pre"]
```

Comment:
You should handle errors as well. That is, if there was an error when executing this, I'm sure `JSON.parse(res.body)["return"].first` won't contain what you are trying to fetch, and so it will raise an exception.